src/main/java/com/nylas/NeuralMessageOptions.java

```diff
@@ -0,0 +1,34 @@
+package com.nylas;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class NeuralMessageOptions {
+	private final Boolean ignore_links;
+	private final Boolean ignore_images;
+	private final Boolean ignore_tables;
+	private final Boolean remove_conclusion_phrases;
+	private final Boolean images_as_markdowns;
```

Comment:
I think there's a typo on this one.  Should be singular.

Reply:
You are correct, good catch.